objects/src/accounts/storage/mod.rs

```diff
@@ -250,7 +291,8 @@ impl Deserializable for AccountStorage {
             items.push((idx, (slot_type, slot_value)));
         }
 
-        Self::new(items).map_err(|err| DeserializationError::InvalidValue(err.to_string()))
+        // ToDo: add correct serialization and deserialization for SMTs
```

Comment:
nit:

```suggestion
        // TODO: add correct serialization and deserialization for SMTs
```

Lots of editors expect the all uppercase version, and tag generation will fail otherwise.

Reply:
Also, this needs the deserialization code.